src/interfaces/swig/shogun.i

```diff
@@ -172,6 +172,13 @@ namespace shogun
 			$self->put(tag_t, value);
 	}
 
+	/* get method for string only available from the interfaces to disambiguate
+	 * it from get_option */
+	std::string get_string(const std::string& name) const
```

Comment:
@karlnapf I think this is the best solution for provide also a `get_string` method from the interfaces.

Reply:
I think this is fine!